docs/devcontainer/viz-tutorial.ipynb

```diff
@@ -0,0 +1,540 @@
+      "\u001b[34mℹ\u001b[39m \u001b[90m｢wdm｣\u001b[39m: Compiled successfully.\n",
+      "\u001b[34mℹ\u001b[39m \u001b[90m｢wdm｣\u001b[39m: Compiling...\n",
+      "\u001b[34mℹ\u001b[39m \u001b[90m｢wdm｣\u001b[39m: Hash: \u001b[1mf44406baf0346ed8ad10\u001b[39m\u001b[22m2m39m\u001b[22mp.js\u001b[39m\u001b[22m\u001b[2K\u001b[2K\u001b[2K\u001b[2K\u001b[2K\u001b[2K\u001b[2K\u001b[2K\u001b[2K\u001b[2K\u001b[2K\u001b[2K\u001b[2K\u001b[2K\u001b[2K\u001b[2K\u001b[2K\n",
+      "Version: webpack \u001b[1m4.47.0\u001b[39m\u001b[22m\n",
+      "Time: \u001b[1m14928\u001b[39m\u001b[22mms\n",
+      "Built at: 11/10/2023 \u001b[1m2:16:46 PM\u001b[39m\u001b[22m\n",
+      "                                     \u001b[1mAsset\u001b[39m\u001b[22m      \u001b[1mSize\u001b[39m\u001b[22m  \u001b[1mChunks\u001b[39m\u001b[22m  \u001b[1m\u001b[39m\u001b[22m                             \u001b[1m\u001b[39m\u001b[22m\u001b[1mChunk Names\u001b[39m\u001b[22m\n",
+      "    \u001b[1m\u001b[32mapp.b687832adda5702a5eee.hot-update.js\u001b[39m\u001b[22m  58.3 KiB     \u001b[1mapp\u001b[39m\u001b[22m  \u001b[1m\u001b[32m[emitted] [immutable] [hmr]\u001b[39m\u001b[22m  app\n",
+      "\u001b[1m\u001b[32mapp.b687832adda5702a5eee.hot-update.js.map\u001b[39m\u001b[22m  92.5 KiB     \u001b[1mapp\u001b[39m\u001b[22m  \u001b[1m\u001b[32m[emitted] [dev]\u001b[39m\u001b[22m              app\n",
+      "                                    \u001b[1m\u001b[32mapp.js\u001b[39m\u001b[22m  45.6 MiB     \u001b[1mapp\u001b[39m\u001b[22m  \u001b[1m\u001b[32m[emitted]\u001b[39m\u001b[22m                    app\n",
+      "                                \u001b[1m\u001b[32mapp.js.map\u001b[39m\u001b[22m  20.7 MiB     \u001b[1mapp\u001b[39m\u001b[22m  \u001b[1m\u001b[32m[emitted] [dev]\u001b[39m\u001b[22m              app\n",
+      "      \u001b[1m\u001b[32mb687832adda5702a5eee.hot-update.json\u001b[39m\u001b[22m  45 bytes        \u001b[1m\u001b[39m\u001b[22m  \u001b[1m\u001b[32m[emitted] [immutable] [hmr]\u001b[39m\u001b[22m  \n",
+      "Entrypoint \u001b[1mapp\u001b[39m\u001b[22m = \u001b[1m\u001b[32mapp.js\u001b[39m\u001b[22m \u001b[1m\u001b[32mapp.b687832adda5702a5eee.hot-update.js\u001b[39m\u001b[22m \u001b[1m\u001b[32mapp.js.map\u001b[39m\u001b[22m \u001b[1m\u001b[32mapp.b687832adda5702a5eee.hot-update.js.map\u001b[39m\u001b[22m\n",
+      "[\u001b[1m./app.js\u001b[39m\u001b[22m] 44.1 KiB {\u001b[1m\u001b[33mapp\u001b[39m\u001b[22m}\u001b[1m\u001b[32m [built]\u001b[39m\u001b[22m\n",
+      "    + 12868 hidden modules\n",
+      "\u001b[34mℹ\u001b[39m \u001b[90m｢wdm｣\u001b[39m: Compiled successfully.\n",
+      "^C\n"
+     ]
+    }
+   ],
+   "source": [
+    "!cd /home/app/viz && . /usr/local/share/nvm/nvm.sh && nvm install 16.13.0 && nvm use 16.13.0 && node --version && npm install && npm start"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "I seem to be stuck... Let's try and capture some more debugging information really quick.\n",
```

Comment:
Still stuck?

Reply:
NPM run cell is what I was working with, and I need to re-edit for ChatGrid functionality to work... Viz notebook is WIP as only viz not chat grid works